Week2/Homework/Grade-calculator.js

```diff
@@ -0,0 +1,48 @@
+ 
+
+  let score=prompt("Enter your score"); // Manual input
+  let full_Score=100;
+
+ // This function turns scores into percentage values.
+ function percentage_calculator (studentScore, maxScore){
+    let studentscore = Math.round(( score / full_Score) * 100);
+    return studentscore;
+}
+
+
+// This function shows the letter grade and score as percentage value.
+
+function grade_calculator (studentscore , studentGrade){
```

Comment:
Perhaps you don't need `studentGrade` as a function parameter here at all. Also, you aren't passing `studentGrade` as an argument but `full_Score` on line 46. It is a bit confusing. 🙂

Reply:
Fixed.